### PR TITLE
Edit Profile Client

### PR DESCRIPTION
**Pull request by phuongthuan**
Story URL: https://trello.com/c/LXDv09Y1/42-2-sprint3-update-user-profile

### CODE REVIEW

**Review by ggianglv — APPROVED** [review]
LGTM

File: src/components/ProfileFormUpdate/index.js [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+      }
+
+      if (error) return <div>Error: {error.message}</div>;
+
+      const { id, avatar, email, address, phone } = data.me;
+
+      return (
+        <Mutation
+          mutation={EDIT_PROFILE_MUTATION}
+          // update={(store, { data: { updateDailyReport } }) => {
```
Comment (queq1890) [inline src/components/ProfileFormUpdate/index.js]: I think it's better to delete line 30 ~ 38 or to write `update` for profile

File: src/components/ProfileFormUpdate/index.js [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+                    <TextInput
+                      type="text"
+                      label="Email"
+                      name="email"
+                      defaultValue={values.email}
+                      error={touched.email && errors.email}
+                      onChange={handleChange}
+                    />
+
+                    <TextInput
```
Comment (queq1890) [inline src/components/ProfileFormUpdate/index.js]: Do we update `avatar`? That means we will need to use some file hosting service.
Like AWS S3 or Cloudinary or something.

**Review by queq1890 — COMMENTED** [review]
Checked & commented.

File: src/components/ProfileFormUpdate/index.js (hunk 2 above)
Comment (phuongthuan) [reply src/components/ProfileFormUpdate/index.js]: Currently, I just allow update avatar by passing URL. We will improve in the future.

File: src/components/ProfileFormUpdate/index.js (hunk 1 above)
Comment (phuongthuan) [reply src/components/ProfileFormUpdate/index.js]: Yeah, a small my mistake. I'll delete that. Thank for reminding !

Comment (queq1890) [reply src/components/ProfileFormUpdate/index.js]: Sorry for annoying, but thanks m(_ _)m

File: src/components/ProfileFormUpdate/index.js (hunk 2 above)
Comment (queq1890) [reply src/components/ProfileFormUpdate/index.js]: Ah OK, I got it!

**Review by queq1890 — APPROVED** [review]

**Pull request merged**